docs/bouwblokken/fundamentele-rechten/non-discriminatie.md

```diff
@@ -25,36 +26,39 @@ Door bij de ontwikkeling van algoritmes rekening te houden met vereisten die voo
  
 De onderwerpen bias en non-discriminatie spelen daarom een belangrijke rol bij de totstandkoming van verantwoord ontwikkelde algoritmen en AI en het gebruik daarvan door ambtenaren. 
 
-### Wat is bias?
-*Bias* is een Engelse term die in het Nederlands wordt vertaald als vooringenomenheid, vooroordeel of neiging. 
+## Wat is discriminatie en bias?
+
+### Discriminatie
+[Artikel 1 van de Nederlandse Grondwet](https://wetten.overheid.nl/jci1.3:c:BWBR0001840&hoofdstuk=1&artikel=1&z=2023-02-22&g=2023-02-22) verbiedt discriminatie: 
+
+> Allen die zich in Nederland bevinden, worden in gelijke gevallen gelijk behandeld. Discriminatie wegens godsdienst, levensovertuiging, politieke gezindheid, ras, geslacht, handicap, seksuele gerichtheid of op welke grond dan ook, is niet toegestaan. 
+
+Non-discriminatie is dus de basis van ons rechtssysteem en onze samenleving. 
```

Comment:
Ik vind het ver gaan om het de basis van ons rechtssysteem te noemen. Weet niet of dat uit externe review kwam. Misschien "De prominente positie in de Grondwet benadrukt het belang van het mensenrecht in Nederland"?

Reply:
Hm ik snap wat je bedoelt. Feedback was om te beginnen bij artikel 1 van de grondwet, eigenlijk net zoals de handreiking non-discriminatie doet. De handreiking verwoort dit ook zo. Iets genuanceerder lijkt me beter!